specification/security/resource-manager/Microsoft.Security/preview/2023-06-01-preview/aggregations.json

```diff
@@ -0,0 +1,193 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Aggregations Query",
+    "description": "API spec for Microsoft.Security (Azure Security Center) Aggregations Query",
+    "version": "2023-06-01-preview"
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "security": [
+    {
+      "azure_auth": [
+        "user_impersonation"
+      ]
+    }
+  ],
+  "securityDefinitions": {
+    "azure_auth": {
+      "type": "oauth2",
+      "authorizationUrl": "https://login.microsoftonline.com/common/oauth2/authorize",
+      "flow": "implicit",
+      "description": "Azure Active Directory OAuth2 Flow",
+      "scopes": {
+        "user_impersonation": "impersonate your user account"
+      }
+    }
+  },
+  "paths": {
+    "/providers/Microsoft.Security/aggregations": {
+      "post": {
+        "tags": [
+          "aggregations"
+        ],
+        "description": "Run pre-defined aggregation query against Microsoft.Security data types for scopes specified in the request.",
+        "operationId": "Aggregations",
+        "parameters": [
+          {
+            "$ref": "../../../common/v1/types.json#/parameters/ApiVersion"
```

Comment:
> "$ref": "../../../common/v1/types.json#/parameters/ApiVersion" [](http://example.com/codeflow?start=11&length=63)

nit, I'd suggest referencing a newer common-types, v3 or later for this

Reply:
There is no other version under security folder, all other apis under security uses v1